agimus_controller/agimus_controller/ocp_base_croco.py

```diff
@@ -51,36 +61,45 @@ def __init__(
             self._running_model_list,
             self._terminal_model,
         )
+        self._problem.nthreads = 6
```

Comment:
This should not be hardcoded, but provided as a parameter to the user through ROS